back-end/package.json

```diff
@@ -5,7 +5,8 @@
   "main": "build/App.js",
   "scripts": {
     "build": "tsc",
-    "start": "node build/App.js"
+    "start": "node build/App.js",
+    "dev": "nodemon --watch 'src/**/*.ts' --exec 'ts-node' src/App.ts"
```

Comment:
`npm run dev` doesn't work on my system because `ts-node` isn't found. I ran `npm install`, but `ts-node` is installed as local package, not global. `npx` runs commands from local packages.
May I suggest changing to:
`"dev" : "dev": "npx nodemon"`
And add a file `nodemon.json` with the contents:
``` json
{
    "watch": ["src"],
    "ext": "ts",
    "exec": "npx ts-node ./src/App.ts"
}
```
Also add a note about the new `npm run dev` command in README.md

Reply:
Yep good call @cskeogh. Just pushed changes, should be good now.